docs/topics/gradle.md

```diff
@@ -747,12 +747,13 @@ If you use [kapt](kapt.md), note that kapt annotation processing tasks are not c
 ## Gradle configuration cache support
 
 > The configuration cache is available in Gradle 6.5 and later as an experimental feature.
+> It is available only on JVM and JS platforms.
```

Comment:
Probably better to say which Gradle plugins support it:
- 'org.jetbrains.kotlin.jvm'
- 'org.jetbrains.kotlin.js'
- 'org.jetbrains.kotlin.android'

Using Platfrom approach suggests MPP/JS is supported, while most probably it may be not

Reply:
Oh, I see!
Thanks for clarification!